connector/datadogconnector/connector.go

```diff
@@ -85,6 +89,9 @@ func (c *connectorImp) Capabilities() consumer.Capabilities {
 
 func (c *connectorImp) ConsumeTraces(ctx context.Context, traces ptrace.Traces) error {
 	c.agent.Ingest(ctx, traces)
+	if c.tracesConsumer != nil {
```

Comment:
Shouldn't the connector mark the traces in some way before calling `ConsumeTraces`?

Reply:
It happens in Ingest method https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/be81ae7675a3d7fb92ed16ff1574a46ceb233db2/internal/datadog/agent.go#L145

Reply:
Thanks for clarifying